sequencer/rpc_endpoint/src/starknet_backend.rs

```diff
@@ -258,19 +247,12 @@ impl StarknetRpcApiServer for StarknetBackend {
         transaction_hash: FeltParam,
     ) -> RpcResult<MaybePendingTransactionReceipt> {
         // necessary destructuring so that we can use a hex felt as a param
```

Comment:
this comment is a little bit out of place now that the destructuring happens inside the `get_transaction_receipt()` call (same with the other RPC call)